catala/sortir/sortir.md

```diff
@@ -1,6 +1,10 @@
 Et poses dempeus i busques una sortida, no veus res.
 De cop i volta, el terra es torna tou i t'enfonses a poc a poc.
 
-Cada cop estàs més enfonsat... de sobte mires a d'alt i veus 
-uns peus que es belluguen amb impaciència... aquestes sabates 
-et sonen... Ets tu!
\ No newline at end of file
+Cada cop estàs més enfonsat... de sobte mires a d'alt i veus
+uns peus que es belluguen amb impaciència... aquestes sabates
+et sonen... Ets tu!
+
+[Fuges espantat](fugir/fugir.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.